fuzz/fuzz_targets/c_rust_merkle.rs

```diff
@@ -78,7 +79,7 @@ mod tests {
     #[test]
     fn duplicate_crash() {
         let mut a = Vec::new();
-        extend_vec_from_hex("d73804000000000000000000000002040404040000000000000040000001ff0000000800380000000000000002040404040000000000000040000001ff0000000800380000000000000000000000000000385cf8ffffff10cdf7bb2cd063000000000000042d343507fffffffffff800404220fffffff800000040420f0000", &mut a);
+        extend_vec_from_hex("00000004ffff", &mut a);
```

Comment:
ac0cb0c2c37bf63b407ea85835eab52f12ac2973

> `=====================================================================
> 0000003238000085000000000010ff07fffffffffffffffff0ffc47758e802cb5de0f1e38c07ee3e5975ebf9043433040404040401c71c71c71c71c80404040494061101000000c7711cc70000000000000004000001000100000000000000040409000000000000001f000000000000`

> C accepted code that Rust rejected: bitstream had trailing bytes 0x00...

Reply:
Ah, it's my responsibility to call closeBitstream in my C-based test harness.